### PR TITLE
Restaurant page feature

### PR DESCRIPTION
**Pull request by ammarkandel**
# Simple restaurant page using javascript (webpack)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
...
+- Twitter: [@AmmarQandel](https://twitter.com/AmmarQandel)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ammar-kandel-7b4100193/)
+
+### Contributing
+
+This is a project for educational purposes only. We are not accepting contributions.
+
+### Show your support
+
+Give a ⭐️ if you like this project!
```
Comment (christianotieno) [inline README.md]: ## Improve the README
- Currently, we do have some missing information that seems to be missing and should be added on here in order to have a more professional and well-documented writeup.
- [ ] Do you mind adding a screenshot of the page here?
- [ ] Add more information of what the project is all about here as well.
- [ ] While at it. I would encourage some improvements on the commit messages when conducting these changes. The commit messages should utmost be reflective and informative of what the changes being introduced are about.
- [ ] Do you mind using this [template](https://github.com/microverseinc/readme-template) format for better inspiration in formating?

File: src/app/components/navbar.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  li.textContent = liContent[i];
+  ul.appendChild(li);
+}
+
+div.classList.add('logo');
+div.textContent = 'Tito';
+
+nav.appendChild(div);
+nav.appendChild(ul);
+document.body.appendChild(nav);
```
Comment (christianotieno) [inline src/app/components/navbar.js]: ## Ensure the page is navigable to the rest of the pages.
- [ ] Please add the functionality to ensure that the tabbed menu, when clicked, the content is replaced and the user gets to see what's on the page. Like in this case, they expect to see the `menu` page when they click the menu tab. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)).

File: webpack.dev.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        test: /\.scss$/,
+        use: [
+          'style-loader',
+          'css-loader',
+          'sass-loader',
+        ],
+      },
+    ],
+  },
+});
```
Comment (christianotieno) [inline webpack.dev.js]: ## Improve the webpack configuration to have the dist directory.
- [ ] The project expects to have a `dist` directory that contains the chunked out minimised code to show on the page. That ought to have been the main purpose of the webpack in this project. As a result, the `dist` dir is expected to have:
- [ ] A main file named `index.html`
- [ ] A `main.js` file that is built by webpack
- [ ] The `index.html` links the `main.js` file as shown below:

```javascript
const myModule = (parentNode) => {
  const node = document.createElement("div");
  // Add whichever sub-nodes, ids and style is required

  parentNode.appendChild(node);
};

export { myModule as default };
```

### Note: 
_The `npm run build` command is not functioning as expected. Hence the detailed review up here_ ☝🏾

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Helllo @ammarkandel 👋🏾 
The design of the page looks great. I loved the design and I feel the extra mile you went to incorporate wepack and the inclusion of the relevant scripts was commendable. Keep it up. 👏🏾 

There are some few suggestions down below that I feel are worth looking at before we proceed. Have a look 👇🏾 

### Required changes 
- [ ] Ensure the page is navigable to the rest of the pages.
- [ ] Improve the webpack configuration to have the `dist` directory.
- [ ] Improve the README 
- [ ] Add more details on the rest of the pages.

### Happy coding 💻 
#### Christian Otieno (CR)

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @ammarkandel :wave:

__I must commend your efforts so far in this project. Here are the major highlights for me:__

- Your Readme looks more professional :100: 
- You've fixed most of the required changes by the last reviewer :+1: 

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] As mentioned by the last reviewer, your application should feature a tabbed menu. When a tab is clicked the content is replaced (<a href="https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png">screenshot</a> from <a href="https://www.theodinproject.com/courses/javascript/lessons/restaurant-page" rel="nofollow noreferrer noopener" target="_blank">Odin</a>)
- You need to ensure that when clicked, the content of the page is replaced with whatever tab is clicked. For instance, when the menu is clicked, the content of the page is replaced with the menu items of the restaurant page.
- [ ] Kindly store the contents of each tab in a <code>module</code>. (<a href="https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png">screenshot</a> from <a href="https://www.theodinproject.com/courses/javascript/lessons/restaurant-page" rel="nofollow noreferrer noopener" target="_blank">Odin</a>)

_**Kindly go through the project specification all over again and ensure that you understand what the tabbed menu feature is all about.**_

Did you need help? You can check out [this article ](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F360026625973-How-can-I-get-help-from-TSEs-) on how to get help from a full-time TSE.

- [ ] Please add the live demo link to the project readme

* * *
**Please make the required changes and submit another review request when you are done.**

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Ammar Kandel** 
Good job with the improvements for this project :muscle:  :muscle:
It meets most of the requirements :clap: :clap: :clap:  
Just to improve before you can go ahead:

- [x] Your application should appends HTML elements to the div with id="content" when the page is loaded.
- [x] The contents of each tab are stored in a module (a different module for each tab).
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi @ammarkandel :wave: 

We are almost good to go. 

Just fix your linter validation, your last commit failed linter validation.

#### That's it, make this change, then submit another form for a code review.


### Have a great day :slightly_smiling_face: 

#### ``` Uchechukwu Inyama Henry ```

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**